components/partner.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const partner = ({name}) => {
```

Comment:
```suggestion
const Partner = ({name}) => {
```

React component names [must start with a capital letter](https://pusher.com/blog/troubleshoot-react-errors/#not-starting-component-names-with-a-capital-letter).